src/Umbraco.Cms.Integrations.Crm.ActiveCampaign/Umbraco.Cms.Integrations.Crm.ActiveCampaign.csproj

```diff
@@ -11,11 +11,12 @@
 		<Title>Umbraco CMS Integrations: CRM - ActiveCampaign</Title>
 		<Description>An extension for Umbraco CMS providing a form picker and rendering component for ActiveCampaign.</Description>
 		<PackageIconUrl></PackageIconUrl>
-		<PackageProjectUrl>https://github.com/umbraco/Umbraco.Cms.Integrations</PackageProjectUrl>
+		<PackageProjectUrl>https://github.com/umbraco/Umbraco.Cms.Integrations/tree/main/src/Umbraco.Cms.Integrations.Crm.ActiveCampaign</PackageProjectUrl>
 		<RepositoryUrl>https://github.com/umbraco/Umbraco.Cms.Integrations</RepositoryUrl>
 		<Version>1.0.0</Version>
 		<Authors>Umbraco HQ</Authors>
 		<Company>Umbraco</Company>
+		<PackageTags>Umbraco-Marketplace</PackageTags>
```

Comment:
Could add "Umbraco" here too.  Not needed for marketplace but it's common to tag Umbraco packages with this in case people are searching via tags on NuGet.